### ISSUE TITLE
OpenSSH lacks manpages

### ISSUE THREAD

**Issue by xeffyr** [termux/termux-packages#1538]
OpenSSH package has manpage only for 'ssh-copy-id'.

```
openssh
└── data
    └── data
        └── com.termux
            └── files
                └── usr
                    ├── bin
                    │   ├── scp
                    │   ├── sftp
                    │   ├── source-ssh-agent
                    │   ├── ssh
                    │   ├── ssh-add
                    │   ├── ssh-agent
                    │   ├── ssh-copy-id
                    │   ├── ssh-keygen
                    │   ├── ssh-keyscan
                    │   ├── ssha
                    │   └── sshd
                    ├── etc
                    │   └── ssh
                    │       ├── moduli
                    │       ├── ssh_config
                    │       └── sshd_config
                    ├── libexec
                    │   ├── sftp-server
                    │   ├── ssh-keysign
                    │   └── ssh-pkcs11-helper
                    ├── share
                    │   └── man
                    │       └── man1
                    │           └── ssh-copy-id.1
                    └── var
                        └── run
                            └── README.openssh

14 directories, 19 files
```

**Comment by Neo-Oli** [comment]
Oh yeah, weird. They seem to have vanished after the 7.5p1-6 version. 7.5p1-5 still had them.

**Comment by fornwall** [comment]
Thanks, fixed in version `7.5p1-8` of the package.

I'm don't know why the package was built without man pages. I suspected that it would happen if overwriting a previous build - if the build system leaved the timestamps of the already installed man pages without touching them, they wouldn't be part of the package. But that doesn't seem to be the case.

Anyway, I've added a test in `termux_step_post_massage` which will fail if the built packages does not contain the man pages, so if it happens again it will be detected at build time.
